dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/org/apache/cxf/jaxb/JAXBDataBinding.java

```diff
@@ -457,13 +520,28 @@ protected void justCheckForJAXBAnnotations(ServiceInfo serviceInfo) {
         }
     }
     private void checkForJAXBAnnotations(MessagePartInfo mpi, SchemaCollection schemaCollection, String ns) {
+
+        if (LOG.isLoggable(Level.FINE)) { // Liberty change begin
+           LOG.fine("checkForJAXBAnnotations for MessagePart:" + (mpi != null ? mpi.getName() : "null") );
+        } // Liberty change end
+
         Annotation[] anns = (Annotation[])mpi.getProperty("parameter.annotations");
+        if (LOG.isLoggable(Level.FINE)) { // Liberty change begin
```

Comment:
Maybe continue using isLoggableFine instead of checking again?

Reply:
yes, will update